config/vufind/KohaRest.ini

```diff
@@ -184,3 +184,14 @@ extraFields = item-issue:acceptTerms:pickUpLocation
 [ItemTypeRenewalBlockMappings]
 ; ILL[too_many] = "ILSMessages::ill_item_renewal_limit"
 ; ILL[too_soon] = "ILSMessages::ill_renewal_too_soon"
+
+; This section allows libraries to define different custom itemLimit rules for 
+; different biblio-level item types.
+; In Koha the biblio-level item type is defined in the 942$c subfield.
+; The string inside the [] is the Koha biblio-level item type.
+; The string after the equal sign is the number of items to display in the holdings tab.
+; By default, all items are displayed in the holdings tab, unless a rule has been defined 
+; for the relevant biblio-level item type for the biblio being loaded.
+[Holdings]
+;itemLimit[J] = 3
```

Comment:
Might it make sense to add a default catch-all item limit, so users can set a single value for all bib levels if they wish? Maybe `itemLimit[DEFAULT]` or `itemLimit[*]` (if the .ini parser will let you get away with that, which it might not).

Reply:
thanks @demiankatz I've added that in my follow-up!